### DISCUSSION TITLE
Filezilla client

### DISCUSSION THREAD

**Discussion by ghost** [swizzin/swizzin#947] [Feature Requests] [answered]
Hi. How about implementing a FileZilla client.

**Comment by brettpetch** [comment, marked as answer]
This is a desktop application and is well outside the scope of the project. If you're looking for a client that is already built in, or that is installable via apt you could use rsync, rclone, or even the ftp cli. Desktop apps aren't something that we're looking to support, as this is primarily meant to be installed on servers.
